.gdb_history

```diff
@@ -0,0 +1,25 @@
+break main
```

Comment:
Remove this file, please.

Reply:
How did this end up there, in the first place?